spyder/plugins/application/container.py

```diff
@@ -278,7 +288,10 @@ def _check_updates_ready(self):
             box.exec_()
             check_updates = box.is_checked()
         else:
+
             if update_available:
+                self.application_update_status.set_status_pending()
+
                 header = _("<b>Spyder {} is available!</b><br><br>").format(
```

Comment:
Can we say something like

```html
<b>Spyder {} is available!</b> <i>(you have {})</i>
```

so the user knows what version of Spyder they will be upgrading _from_? This is very common and often expected on update dialogs.